controller/work-item-link-blackbox_test.go

```diff
@@ -264,7 +264,7 @@ func CreateWorkItemLinkCategory(name string) *app.CreateWorkItemLinkCategoryPayl
 }
 
 // CreateWorkItem defines a work item link
-func CreateWorkItem(workItemType string, title string) *app.CreateWorkitemPayload {
+func CreateWorkItem(workItemType satoriuuid.UUID, title string) *app.CreateWorkitemPayload {
```

Comment:
We have a mixture of `satoriuuid` and `uuid` and we should make that consistent at one point. But in this code it was imported as `satoriuuid` and that's why I've used what was already there.